File: JSBASIC/124_gcd.js
// 124	124_gcd.js	Find the GCD (Greatest Common Divisor) of two numbers using a loop.

let a = 48, b = 18;
let x = a, y = b;
while (y !== 0) {
    let temp = y;
    y = x % y;
    x = temp;
}
console.log("Numbers:", a, "and", b);
console.log("GCD:", x);

File: JavaBasic/124_gcd.java
// 124	124_gcd.java	Find the GCD (Greatest Common Divisor) of two numbers using a loop.
public class _124_gcd {
    public static void main(String[] args) {
        int a = 48, b = 18, x = 48, y = 18;
        while (y != 0) { int t = y; y = x%y; x = t; }
        System.out.println("GCD of " + a + " and " + b + " = " + x);
    }
}
